output.tf

```diff
@@ -25,3 +25,7 @@ output "db_subnet_rules" {
 output "name" {
   value = azurerm_template_deployment.postgres-paas.name
 }
+
+output "resource_group_name" {
```

Comment:
can you also create a PR against the https://github.com/hmcts/cnp-module-postgres/tree/postgresql_tf branch please?

Reply:
Yeah, no problem